src/CovidData/covidData.ts

```diff
@@ -39,3 +39,13 @@ const percentDifference = (a: number, b: number) => {
   const result = Math.round(((a - b) / b) * 100)
   return result
 }
+
+type TrendData = number[]
+
+export const toLineChartCasesPer100k = (data: CovidData): TrendData => {
+  return data.map(toCasesNewPer100k).slice(0, 7)
+}
+
+export const toCasesNewPer100k = (datum: CovidDatum): number => {
+  return datum.positiveCasesNew / 100000
```

Comment:
I believe this is incorrect, it should be 100k residents of the population.

Reply:
good catch, I suppose we'll need to pass the HA population in through the `env`, and the calculation would be `datum.positiveCasesNew` / `totalPopulation`/ `100000`? 🤔

Reply:
Yup, i think this makes sense to be coming from the api, we dont want to be keeping track of the populations.  https://covidactnow.org/ looks like it has this available directly.